pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2009,7 +2008,7 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.0.0",
         "4.0.10.0": "4.0.10",
-        "4.1.0.0": "4.0.11",
+        "4.1.0.0": "4.3.0",
```

Comment:
No, sort is not enforced.  That's just coincidence.  JSON.Net is just serializing a dictionary.